lib/streamlit/elements/text.py

```diff
@@ -25,14 +25,22 @@
 
 class TextMixin:
     @gather_metrics("text")
-    def text(self, body: "SupportsStr") -> "DeltaGenerator":
+    def text(
+        self,
+        body: "SupportsStr",
+        *,  # keyword-only arguments:
+        help: Optional[str] = None,
+    ) -> "DeltaGenerator":
         """Write fixed-width and preformatted text.
 
         Parameters
         ----------
         body : str
             The string to display.
 
+        help : str
+            An optional tooltip that gets displayed next to the widget.
```

Comment:
```suggestion
            An optional tooltip that gets displayed next to the text.
```

`st.text` is a static element, not a widget. The docstring should say "... next to the text".